docs/openapi.yaml

```diff
@@ -0,0 +1,1143 @@
+          example: 25.4
+        xMax:
+          type: number
+          format: float
+          description: Maximum x-coordinate in asset frame covered by map
+          example: 254.3
+        yMin:
+          type: number
+          format: float
+          description: Minimum y-coordinate in asset frame covered by map
+          example: -10.3
+        yMax:
+          type: number
+          format: float
+          description: Maximum x-coordinate in asset frame covered by map
+    Stream:
+      type: object
+      properties:
+        protocol:
+          type: string
+          description: Protocol used for streaming real time data
+          enum: ["MQTT"]
+        host:
+          type: string
+          format: ipv4
+          description: IP-address of host
+          example: "192.0.2.146"
+        port:
+          type: string
+          description: Port number used
+          example: "1888"
+        topics:
+          type: array
+          items:
+            $ref: "#/components/schemas/Topic"
+    Topic:
+      type: object
+      properties:
+        path:
+          type: string
+          example: "robot/pose"
+        resource:
+          type: string
+          enum: ["battery", "pose"]
+    Schedule:
+      type: object
+      properties:
+        robotId:
+          type: string
+          format: uuid
+          description: Robot id for schedule
+          example: "1b69b353-95ff-402e-9864-a5acf5bd9ddb"
+        startDate:
+          type: string
+          format: date
+          description: Start date for schedule
+          example: "2022-01-12"
+        endDate:
+          type: string
+          format: date
+          description: End date for schedule
+          example: "2022-01-19"
+        events:
+          type: array
+          items:
+            $ref: "#/components/schemas/Event"
+    Event:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Id if schedule entry
+          example: "1b69b353-95ff-402e-9864-a5acf5bd9ddb"
+        seriesId:
+          type: string
+          format: uuid
+          description: Series id if the schedule entry is a part of a series
+          example: "1b69b353-95ff-402e-9864-a5acf5bd9ddb"
+          nullable: true
+        missionId:
+          type: integer
+          format: int32
+          description: Id of mission in echo
+          example: 231
+        startTime:
+          type: string
+          format: date-time
+          description: Start time for scheduled mission
+          example: "2022-01-12T10:33:33Z"
+        endTime:
+          type: string
+          format: date-time
+          description: Expected end time for scheduled mission
+          example: "2022-01-13T10:33:33Z"
+    InspectionType:
+      type: string
+      description: Valid inspection types
+      enum: ["thermal_image", "image", "audio"]
```

Comment:
Should `InspectionType`, `ReportStatus`, and `RobotStatus` enums be reviewed and aligned with the enums in the database pr #32?